utils/extend_model.py

```diff
@@ -30,6 +30,8 @@ def add_project_name_to_time_entries(time_entries, projects):
             if time_entry.project_id == project.id:
                 name = project.name + " (archived)" if project.is_deleted() else project.name
                 setattr(time_entry, 'project_name', name)
+                setattr(time_entry, 'customer_id', project.customer_id)
```

Comment:
This changes are ok. I think we should change the name of the function **from** `add_project_name_to_time_entries(time_entries, projects)` **to** `add_project_info_to_time_entries(time_entries, projects)`.

Beside of the current function. You just have to update the following lines. 
https://github.com/ioet/time-tracker-backend/blob/93649c100ddb6939394c7b1797f2243ea1cd7e27/time_tracker_api/time_entries/time_entries_model.py#L22
 and
https://github.com/ioet/time-tracker-backend/blob/93649c100ddb6939394c7b1797f2243ea1cd7e27/time_tracker_api/time_entries/time_entries_model.py#L226